pandas/core/arraylike.py

```diff
@@ -357,7 +357,7 @@ def reconstruct(result):
         # * len(inputs) > 1 is doable when we know that we have
         #   aligned blocks / dtypes.
         inputs = tuple(np.asarray(x) for x in inputs)
-        result = getattr(ufunc, method)(*inputs)
+        result = getattr(ufunc, method)(*inputs, **kwargs)
```

Comment:
the else clause is reached for a DataFrame when not `(len(inputs) > 1 or ufunc.nout > 1)`.

The `result = mgr.apply(getattr(ufunc, method))` there for `__call__` also fails to pass along **kwargs

Is it straightforward to add to the paramterised here to exercise that path.

Reply:
Yep, I think so...will let you know if I run into any issues

Reply:
Added a test, but had to xfail because `out` is not written to correctly. The written result becomes transposed with the block-wise apply, I'm guessing because of the following relationship:
```
arr = np.array([[1, 2], [3, 4]])
df = pd.DataFrame(arr)
print(df.values)
print(df._mgr.blocks[0].values)
```
gives 
```
[[1 2]
 [3 4]]
[[1 3]
 [2 4]]
```

Reply:
Yeah, basically when `out` is specified, I think we should simply not call `mgr.apply`, see https://github.com/pandas-dev/pandas/issues/39275, https://github.com/pandas-dev/pandas/pull/39260/files for a recent similar case where certain additional keyword arguments cannot be handled on a block-by-block case.